webplatformcompat/cache.py

```diff
@@ -176,11 +185,35 @@ def feature_v1_loader(self, pk):
 
     def feature_v1_add_related_pks(self, obj):
         """Add related primary keys to a Feature instance."""
+        def row_descendant_pks(row_children):
+            """Recursively gather row descendants."""
+            row_pks = []
+            for child in row_children:
+                if not child.mdn_uri:
+                    row_pks.append(child.pk)
+                    sub_row_children = child.children.only('pk', 'mdn_uri')
+                    row_pks.extend(row_descendant_pks(sub_row_children))
+            return row_pks
+
         if not hasattr(obj, '_history_pks'):
             obj._history_pks = list(
                 obj.history.all().values_list('history_id', flat=True))
-        if not hasattr(obj, '_children_pks'):
-            obj._children_pks = list(obj.children.values_list('pk', flat=True))
+        if not (hasattr(obj, '_children_pks') and
+                hasattr(obj, '_row_children_pks') and
+                hasattr(obj, '_page_children_pks') and
+                hasattr(obj, '_row_descendant_pks')):
+            obj._children_pks = []
+            obj._row_children_pks = []
+            obj._page_children_pks = []
+            row_children = []
+            for child in obj.children.only('pk', 'mdn_uri'):
+                obj._children_pks.append(child.pk)
+                if child.mdn_uri:
```

Comment:
The [Including Child Pages](https://github.com/mdn/browsercompat/pull/65/files#diff-37ff62fc806144a51957c37e625c7dc4R80) doc section in this PR seems fine.